histomicstk/FeatureExtractionHaralick.py

```diff
@@ -0,0 +1,106 @@
+import skimage.feature
+import numpy as np
+
+
+def FeatureExtractionHaralick(I, Dst=1, Ng=256):
+    """
+    Calculates 26 Haralick features from an image
+    Parameters
+    ----------
+    I : array_like
+        An intensity image
+    Dst : Distance of neighboring pixels in an image. Default value = 1.
+    Ng : Number of gray level co-occurance matrix. Default value = 256.
+    Returns
+    -------
+    H : array_like
+        A matrix containing mean and ranges for 26 features
+    References
+    ----------
+    .. [1] Haralick, et al. "Textural features for image classification,"
+    IEEE Transactions on Systems, Man, and Cybernatics, vol. 6, pp: 610-621,
+    1973.
+    .. [2] Luis Pedro Coelho. "Mahotas: Open source software for scriptable
+    computer vision," Journal of Open Research Software, vol 1, 2013.
+    """
+    # sets the number of angles: 0 45 90 135. this must not be modified.
+    NumberofAngles = 4
```

Comment:
Haralick feature uses 4 angles to compute means and ranges of the features. If we need to change the number of angles, it would not be haralick features I guess.